ui/component/or-app/src/index.ts

```diff
@@ -15,15 +15,15 @@ import "./or-header";
 import "@openremote/or-icon";
 import {updateMetadata} from "pwa-helpers/metadata";
 import i18next from "i18next";
-import manager, {Auth, DefaultColor2, DefaultColor3, DefaultColor4, ManagerConfig, Util, BasicLoginResult, normaliseConfig, Manager} from "@openremote/core";
+import manager, {DefaultColor2, DefaultColor3, DefaultColor4, Util, BasicLoginResult, normaliseConfig, Manager} from "@openremote/core";
 import {DEFAULT_LANGUAGES, HeaderConfig} from "./or-header";
 import {OrMwcDialog, showErrorDialog, showDialog} from "@openremote/or-mwc-components/or-mwc-dialog";
 import {OrMwcSnackbar} from "@openremote/or-mwc-components/or-mwc-snackbar";
 import {AnyAction, Store, Unsubscribe} from "@reduxjs/toolkit";
 import {AppStateKeyed, updatePage, updateRealm} from "./app";
 import { InputType, OrInputChangedEvent } from "@openremote/or-mwc-components/or-mwc-input";
 import { ORError } from "@openremote/core";
-import { Realm } from "@openremote/model";
+import { Auth, ManagerAppConfig, ManagerConfig, Realm } from "@openremote/model";
```

Comment:
`ManagerAppConfig` shouldn't be needed in here

Reply:
It is. Because I need a way to hold onto the original settings before they are separated in other settings. Some settings of the `ManagerAppConfig` are used in `ManagerConfig` but not all. 

I found it not logical that the `manager_config.json` is loaded twice. With every page visited it's loaded. And when I don't use this it will be needed to load a second time when a user is on the config page.